src/styles/theme.tsx

```diff
@@ -6,6 +6,7 @@ export const COLORS = {
   primary: "#3A4276",
```

Comment:
@Chegemartinx @Allen-Muhani @Alphonce-Mutebi 

same here, i cannot find any color here that properly matches color + naming in Figma
we need to get this right now, once and for all.
Otherwise we can never add another dev to the team.

We have to systems, Figma and GitHub.
Both need to be exactly aligned in terms of:
1. Folder structure
2. Naming of everything

Where do you need more clarification? where can I help?

Reply:
This file is useless. Im supposed to create a separate PR to remove it.
https://github.com/Kukuza/mobile-wallet/issues/19